recipe/meta.yaml

```
@@ -6,52 +6,81 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 62423895b866f95b5ed8d88335295a37cc5403af64cb7cb0e234f88adc2dff94
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 62423895b866f95b5ed8d88335295a37cc5403af64cb7cb0e234f88adc2dff94
+  - url: https://github.com/aio-libs/aiokafka/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: d71e708393708d00524bd522caabde8b3b488c6fd6e8b96662a60510c22d9f99
+    folder: gh_src
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython >=3.0.5                         # [build_platform != target_platform]
-    - zlib                                   # [build_platform != target_platform]
     - {{ compiler('c') }}
-    - {{ stdlib("c") }}
+    # - {{ stdlib('c') }}
   host:
     - cython >=3.0.5
     - python
     - pip
-    - setuptools
+    - setuptools >=77
+    - wheel
```

wheel is no longer required for setuptools >=70.1.0

```suggestion
    - setuptools >=77
```

---

oh good to know